argus/backends/local/local_backend.py

```diff
@@ -0,0 +1,46 @@
+from argus.backends import base as base_backend
+from argus.backends import windows as windows_backend
+from argus.client import windows
+from argus import config as argus_config
+from argus import log as argus_log
+from argus import util
+
+CONFIG = argus_config.CONFIG
+
+
+class LocalBackend(windows_backend.WindowsBackendMixin,
+                   base_backend.BaseBackend,
```

Comment:
Nope, just meant that it would be okay to have both classes from windows_backend under each other(like your formatting plus a third line for the base_backend), just seemed more readable to me that's all.